### PR TITLE
Updates to pyproject.toml

### PR DESCRIPTION
**Pull request by alalazo**

### CODE REVIEW

**Comment by alalazo** [comment]
@tgamblin have a look at the Trove classifiers to see if there's anything that needs to be modified. For instance I classified the library as `"Development Status :: 3 - Alpha"` but you might have different opinions.

File: pyproject.toml [hunk 1]
```diff
@@ -3,7 +3,14 @@ name = "archspec"
 version = "0.1.0"
 description = "A library to query system architecture"
 license = "MIT"
-authors = ["Massimiliano Culpo <massimiliano.culpo@gmail.com>"]
+authors = ["archspec developers <maintainers@spack.io>"]
+maintainers = [
+"Greg Becker <maintainers@spack.io>",
+"Massimiliano Culpo <massimiliano.culpo@gmail.com>",
+"Todd Gamblin <maintainers@spack.io>"
```
Comment (boegel) [inline pyproject.toml]:
```suggestion
"Todd Gamblin <maintainers@spack.io>"
"Kenneth Hoste <kenneth.hoste@ugent.be>"
```

**Comment by boegel** [comment]
> @tgamblin have a look at the Trove classifiers to see if there's anything that needs to be modified. For instance I classified the library as `"Development Status :: 3 - Alpha"` but you might have different opinions.

tagging it as `Alpha` for now makes sense to me...

**Comment by alalazo** [comment]
@tgamblin Done

**Pull request merged**